src/code/z_elf_message.c

```diff
@@ -165,10 +165,10 @@ u16 ElfMessage_GetSariaText(PlayState* play) {
     return ElfMessage_GetTextFromMsgs(msgs);
 }
 
-u16 ElfMessage_GetCUpText(PlayState* play) {
-    if (play->cUpElfMsgs == NULL) {
+u16 ElfMessage_GetNaviText(PlayState* play) {
+    if (play->naviMsgs == NULL) {
         return 0;
     } else {
-        return ElfMessage_GetTextFromMsgs(play->cUpElfMsgs);
+        return ElfMessage_GetTextFromMsgs(play->naviMsgs);
```

Comment:
If we're moving from `ElfMsgs` to `naviMsgs`, could that also apply to function prefixes in `z_elf_message.c`?
`ElfMessage_GetTextFromMsgs`
to
`NaviMessage_GetTextFromMsgs`?

Reply:
QuestMessage or HintMessage could replace ElfMessage as the system name? Can do that in this PR